src/kibit/driver.clj

```diff
@@ -30,20 +31,22 @@
   (sort-by #(.getAbsolutePath ^File %)
            (filter clojure-file? (file-seq dir))))
 
-(defn run [source-paths & args]
+(defn run [source-paths rules & args]
   (let [[options file-args usage-text] (apply (partial cli args) cli-specs)
         source-files (mapcat #(-> % io/file find-clojure-sources-in-dir)
                              (if (empty? file-args) source-paths file-args))]
-    (mapcat (fn [file] (try (check-file file :reporter (name-to-reporter (:reporter options)
-                                                                         cli-reporter))
-                            (catch Exception e
-                              (println "Check failed -- skipping rest of file")
-                              (println (.getMessage e)))))
+    (mapcat (fn [file] (try (check-file file
+                                       :reporter (name-to-reporter (:reporter options)
+                                                                   cli-reporter)
+                                       :rules (or rules all-rules))
+                           (catch Exception e
```

Comment:
The previous catch block was indented inline with the start of `(check-file`. So:

```
(try (check-file ...)
    (catch Exception e ...))